homeassistant/components/qvr_pro/camera.py

```diff
@@ -90,6 +90,13 @@ def device_state_attributes(self):
 
     def camera_image(self):
         """Get image bytes from camera."""
+        try:
+            return self._client.get_snapshot(self.guid)
+
+        except QVRResponseError as ex:
+            _LOGGER.error(ex)
```

Comment:
Include what you were trying to do when you got the error.

```suggestion
            _LOGGER.error("Error getting image: %s", ex)
```